src/plugins/kibana_usage_collection/server/collectors/management/telemetry_management_collector.ts

```diff
@@ -19,8 +19,13 @@
 
 import { IUiSettingsClient } from 'kibana/server';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { stackManagementSchema } from './schema';
 
-export type UsageStats = Record<string, any>;
+export interface UsageStats extends Record<string, boolean | number | string> {
+  // We don't support `type` yet. Only interfaces. So I added at least 1 known key to the generic
```

Comment:
The tech debt is reasonable here. We can handle it together with any other larger changes we need to make.